client/src/document/forms/edit.scss

```
@@ -1,6 +1,25 @@
 .document-edit {
-  h2 .close {
-    font-size: 70%;
-    margin-left: 20px;
+  .edit-header {
+    .close {
+      font-size: 70%;
+      margin-left: 20px;
+    }
```

Here again, we should use the available variables:

```suggestion
    .close {
      font-size: $tiny-text;
      margin-left: $base-spacing;
    }
```

Also, this is the second place this code is added so, if we decide to stick with this long term, we should move it to minimalist as a utility class.